oshdb-util/src/test/java/org/heigit/bigspatialdata/oshdb/util/geometry/osmtestdata/OSHDBGeometryBuilderTestOsmTestData7xxTest.java

```diff
@@ -1109,17 +1113,20 @@ public void test784() throws ParseException {
 
   @Test
   public void test785() throws ParseException {
-    // https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/oshdb/issues/131
+    /*
+    785 is not really a valid test case.
+
+    According to https://wiki.openstreetmap.org/wiki/Relation:multipolygon: “An implementation of
+    multipolygons should attempt to render [touching inner rings] as if the touching rings were
+    indeed one ring.”
+
+    Here, the inners form more than one ring: one inner ring and one additional outer ring.
+    */
+    // https://github.com/GIScience/oshdb/issues/129
     // Valid OSM multipolygon with two touching inner rings leaving an empty area.
     OSMEntity entity = testData.relations().get(785900L).get(0);
     Geometry result = OSHDBGeometryBuilder.getGeometry(entity, timestamp, tagInterpreter);
-    assertTrue(result instanceof MultiPolygon);
-    assertTrue(result.isValid());
-    assertEquals(2, result.getNumGeometries());
-    assertEquals(1,
-        ((Polygon)result.getGeometryN(0)).getNumInteriorRing()
-        + ((Polygon)result.getGeometryN(1)).getNumInteriorRing()
-    );
```

Comment:
Why is the check for multiple interior rings not necessary anymore?

Reply:
similar argument as above: we don't try to magically fix the invalid input data anymore, so there's no guarantee in the outcome of such input data.